04LessonFour/in-class-project/index.html

```diff
@@ -2,64 +2,71 @@
 <html>
 
 <head>
-    <title>Intel Processor/title>
+    <title>Intel Processor</title>
         <link rel="stylesheet" href="css/style.css">
 </head>
-<this is a test></this>
 
 <body>
     <header>
-        <h1> Intel</h1>
+      <br/>
 
+        <h1> Intel</h1>
+<br/>
         <nav>
-            <a href="#intelcore">i7 Processor</a>
-            <a href="#solid_state">Solid-State Drives</a>
+            <a class=nav href ="#_Intel_Customer_Support">i7 Processor</a>
+            <a class=nav href="#_SolidState">Solid-State Drives</a>
         </nav>
+        <br/>
+
     </header>
 
     <section id="Intel_Core">
-        <h3>Intel &reg; Core &trade; i7</h3>
+        <h3>Intel&reg; Core&trade; i7</h3>
 
         <p>
-            The 6th Generation Intel &reg; Core &Trade; i7 processors deliver a new class of computing with a host of new features to power the next generation of desktops, laptops, and 2 in 1 PCs. Expect lightening fast speeds and peak performance through even the
+            The 6th Generation Intel&reg; Core&Trade; i7 processors deliver a new class of computing with a host of new features to power the next generation of desktops, laptops, and 2 in 1 PCs. Expect lightening fast speeds and peak performance through even the
             toughest of tasks and games.
         </p>
     </section>
 
-    <section>
+    <section id=ready>
         <h4> Ready. Wake. Create.
       </h4>
         <p>
-            From zero to power in a flash, the 6th Generation Intel &reg; Core &Trade; processor wakes quicker than ever and lets you switch in and out of apps more seamlessly than ever
+            From zero to power in a flash, the 6th Generation Intel&reg; Core&Trade; processor wakes quicker than ever and lets you switch in and out of apps more seamlessly than ever
 
         </p>
+      </section>
 
-        <aside>
-            <!--Figure this out-->
-            <img src=< /aside>
+        <aside id="aside_computer">
 
-    </section>
+            <img src="img/processor.jpg" </>
```

Comment:
be on the lookout for auto-complete stuff. not sure if you meant to, but there is an open and close tag at the end of this img ( </> ). also, it's a good idea to alt and title your images.